media/js/src/wizard.js

```diff
@@ -1,15 +1,52 @@
 ;(function() {
 
+    var allRadioButtonsSelected = function(radios) {
+        var groups = {};
+        radios.each(function(a) {
+            groups[$(this).attr('name')] = 1;
+        });
+        var total = Object.keys(groups).length;
+        var selected = radios.filter(function(index) {
+            return $(this).is(':checked');
+        });
+        return total == selected.size();
```

Comment:
Should use `===` here instead of `==`

Also, I hadn't seen `.size()` before, so I looked it up. It's just a minor note, but
I think it this be changed to `.length`, since it's deprecated in jQuery as of v1.8:
https://api.jquery.com/size/